docs/cli-reference.md

```diff
@@ -16,8 +18,8 @@ nav_order: 5
 }
 </style>
 
-* TOC
-{:toc .toc-h2-only}
+- TOC
```

Comment:
Question, did you use the command that I created for this? This change shouldn't be here at all

Reply:
https://github.com/x2ansible/x2a-convertor/blob/main/generate_cli_docs.py